.circleci/config.yml

```diff
@@ -42,9 +42,9 @@ jobs:
       - restore_cache:
           keys:
             - v4-cargo-cache-{{ arch }}-{{ checksum "Cargo.lock" }}
-      - run:
-          name: Check format
-          command: rustup component add rustfmt; cargo fmt --all -- --check
+      # - run:
```

Comment:
Why is the format check removed in this PR?

Reply:
because https://rust-lang.github.io/rustup-components-history/ again 
Maybe it would make sense to ignore this check if the component isn't available, it's not blocking and would prevent these kind of changes..

Reply:
Fixed